### PR TITLE
Add set of passing Tpetra tests

### PR DESCRIPTION
**Pull request by jpthiele**
Another subset of 16 tests from #16611 which are working with the current status of master.
Again, as in #16647 to avoid regression while continuing work on the Tpetra wrappers.

### CODE REVIEW

**Review by bangerth — APPROVED** [review]
Great, thanks for figuring out which ones these were!

**Review by kronbichler — APPROVED** [review]

File: tests/trilinos_tpetra/08.cc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
```
Comment (kronbichler) [inline tests/trilinos_tpetra/08.cc:14]: This and the other tests still have the old headers. Would you please replace this by the current version https://github.com/dealii/dealii/blob/add8e16ae6718e1e0521653ec1f5c1e48db5ee57/tests/base/aligned_vector_01.cc#L1-L13 (just picking a random file here).

Comment (jpthiele) [reply tests/trilinos_tpetra/08.cc]: Done!

**Review by bangerth — APPROVED** [review]

**Pull request merged**